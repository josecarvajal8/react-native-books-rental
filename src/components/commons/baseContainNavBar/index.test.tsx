import React from 'react';
import renderer from 'react-test-renderer';
import BaseContainNavBar from '.';
const mockProps = {
    title: 'test',
    onBack: () => null
}
test('Render booksList component correctly', () => {
    const bookListCmp = renderer.create(<BaseContainNavBar {...mockProps} />).toJSON();
    expect(bookListCmp).toMatchSnapshot();
})