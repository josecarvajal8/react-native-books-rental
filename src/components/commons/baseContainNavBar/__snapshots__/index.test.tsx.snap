// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render booksList component correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "marginTop": 7.444196428571429,
    }
  }
>
  <RNGestureHandlerButton
    collapsable={false}
    onGestureEvent={[Function]}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    onHandlerStateChange={[Function]}
    rippleColor={0}
  >
    <View
      accessible={true}
      style={
        Object {
          "borderWidth": 1.8115942028985508,
          "height": 29.776785714285715,
          "marginLeft": 36.231884057971016,
          "opacity": 1,
          "width": 36.231884057971016,
        }
      }
    >
      <View />
    </View>
  </RNGestureHandlerButton>
  <Text
    forwardedRef={[Function]}
    style={
      Object {
        "marginLeft": 253.62318840579712,
      }
    }
  >
    test
  </Text>
</View>
`;
